### ISSUE TITLE
Will Antifier work with other trainer softare?

### ISSUE THREAD

**Issue by kklasman** [john-38787364/antifier#19]
Since this software converts TACX Fortius signals to Ant+, seems like it should be general purpose and work with other trainer software, such as TrainerRoad. 

Thanks,

Kevin

**Comment by haikom** [comment]
Yes, because Antifier converts to ANT+ FE-C, all compatible trainer software should work. I use Antifier + Tacx iFlow + Zwift. Is is working great and stable - thanks a lot for sharing this piece of code!

Kind regards
Haiko
